### PR TITLE
Threading, notifications, CVSSv3

### PR DESCRIPTION
**Pull request by m-1-k-3**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Feature, bug fixing

* **What is the current behavior?** (You can also link to an open issue here)

Threading needs to be configured per module
CVSSv2 results
Missing notifications

* **What is the new behavior (if this is a feature change)? If possible add a screenshot.**

Threading can now configured centrally, if a module needs special threading we can use local var in the module to overwrite the global setting
Moved from CVSSv2 to v3 and update f20 to handle the json output from cve-search
basic notification system via inotify is now in place

### CODE REVIEW

File: modules/S99_grepit.sh [hunk 1]
```diff
@@ -50,7 +50,7 @@ S99_grepit() {
   local GREPIT_RESULTS=0
 
   local MAX_THREADS_S99
-  MAX_THREADS_S99=$((2*"$(grep -c ^processor /proc/cpuinfo || true )"))
+  MAX_THREADS_S99=$((1*"$(grep -c ^processor /proc/cpuinfo || true )"))
```
Comment (BenediktMKuehne) [inline modules/S99_grepit.sh:53]: is this consistent?
other modules use local MAX_MOD_THREADS

**Review by BenediktMKuehne — COMMENTED** [review]
looks good, just one Q.
Tests running

File: modules/S99_grepit.sh (hunk 1 above)
Comment (m-1-k-3) [reply modules/S99_grepit.sh:53]: good point ... will change this

**Comment by p4cx** [comment]
Code looks good to me - test is still running - if there is nothing bad happening, then I will merge it! Thank you for your nice work!

**Review by p4cx — APPROVED** [review]

**Pull request merged**